src/components/sbb-link/sbb-link.tsx

```diff
@@ -17,43 +17,56 @@ import { i18nTargetOpensInNewWindow } from '../../global/i18n';
 export class SbbLink {
   /**
    * If set to true, the browser will
-   * show the download dialog on click.
+   * show the download dialog on click (optional).
    */
   @Prop() public download?: boolean;
 
-  /** The href value you want to link to */
-  @Prop() public hrefValue!: string;
+  /** The href value you want to link to (mandatory otherwise the link becomes a button)*/
+  @Prop() public href!: string;
```

Comment:
Nitpick: This is not generally mandatory, just for the case when it `should` render as an ancher element.

Reply:
@kyubisation Do we really want to change hrefValue into href? There are a lot of components which use hrefValue.

Reply:
How do you guys want it now (@kyubisation)?^^ Sorry I don't get it fully from the comments.

Reply:
@HendrikExtSBB Sorry for the confusion, it should be "href" as @kyubisation previously defined and you already applied.